babel/src/test/resources/sql/big-query.iq

```diff
@@ -1012,9 +1012,9 @@ FROM
 
 SELECT
```

Comment:
Nit: could we change the commit message to "Use explicit casting if element type in ARRAY/MAP does not equal derived component type"

Reply:
thanks, both updated the Jira and commit name to this.